examples/ContractTestUtils.purs

```diff
@@ -84,7 +88,7 @@ mkAssertions
        )
 mkAssertions params@(ContractParams p) = do
   senderAddress <-
-    liftedM "Failed to get sender address" getWalletAddress
+    Helpers.liftedHead "Failed to get sender address" getWalletAddress
```

Comment:
I'm against using these helpers. The code in the examples must be copy-paste-able, but we don't have these helpers in the public interface.

Some of them are re-exported from prelude, but not all - perhaps we should add these missing ones there.

Reply:
I defined them in `examples/Helpers.purs`, but since I'm also using them in the tests, moving them to `Interanal/Helpers.purs` sounds better.